app/mailers/added_review_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class AddedReviewMailer < ApplicationMailer
+
+  def public_review_notification(public_review)
+    @public_review = public_review
+    reviewer_emails = Person.reviewers.pluck(:email)
```

Comment:
Nie - wszyscy recenzenci nie mają być informowani. To nie m sensu. Wiadomość powinna być wysyłana wyłącznie do redaktora prowadzącego dane zgłoszenie.